addons/spreadsheet/static/src/global_filters/components/filter_numeric_value/filter_numeric_value.js

```diff
@@ -0,0 +1,11 @@
+/** @ts-check */
```

Comment:
change name of file to `numeric_filter_value.js` to keep consistency with the component name